src/js/components/Factoid.js

```diff
@@ -0,0 +1,49 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import Paragraph from './Paragraph';
+
+const CLASS_ROOT = 'factoid';
+
+export default class Factoid extends Component {
+  render() {
+    let classes = [CLASS_ROOT];
+    let IconBefore = undefined;
+    let IconAfter = undefined;
+
+    if (this.props.size) {
```

Comment:
can we use classnames function here?

See this for an example: https://github.com/grommet/grommet/blob/master/src/js/components/Anchor.js#L32